server/src/test/java/org/opensearch/gateway/PrimaryShardBatchAllocatorTests.java

```diff
@@ -85,61 +81,60 @@ private void allocateAllUnassignedBatch(final RoutingAllocation allocation) {
         batchAllocator.allocateUnassignedBatch(shardsToBatch, allocation);
     }
 
-    public void testMakeAllocationDecisionDataFetching() {
```

Comment:
Can we put back the existing tests ? Lets change the method calls of the batchAllocator accordingly

Reply:
Added.